packages/babel-parser/test/fixtures/experimental/module-attributes/invalid-syntax-with-no-attributes-identifier/input.js

```diff
@@ -0,0 +1,2 @@
+import "x"
+with ({});
```

Comment:
This test case is not testing `import-with-no-attributes-identifier` but `withClause: [noLineTerminator] with type: StringLiteral`. Can you rename the test to `import-with-statement`?

We can add a new test for `invalid-syntax-with-no-attributes-identifier`
```js
import "x" with;
```